lib/stm32/common/rtc_common_g4.c

```diff
@@ -0,0 +1,217 @@
+/** @addtogroup rtc_file RTC peripheral API
+@ingroup peripheral_apis
+
+@author @htmlonly &copy; @endhtmlonly 2012 Karl Palsson <karlp@tweak.net.au>
+
+*/
+
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2012 Karl Palsson <karlp@tweak.net.au>
```

Comment:
if this is actually my old file, then it shouldn't be in a new file.  If it's a new file, with new functions, it's not copyright karl, in 2012.